src/map/index.js

```diff
@@ -9,23 +9,31 @@ import view from './view';
  */
 function getLocationFromPlaceString(placeString) {
 	let boxRange = 100000;
-	let lat, lon = null;
+	let lat,
```

Comment:
Do you want to initialize lat with null too?

Reply:
in this case it doesn't matter, but yes, it should be unified

Reply:
https://github.com/gisat-panther/ptr-utils/pull/15/commits/a803899931945a96afa6ecee53a4f4637474da7a